ibis/expr/operations.py

```diff
@@ -3623,23 +3624,17 @@ class GeoAsText(GeoSpatialUnOp):
 class ElementWiseVectorizedUDF(ValueOp):
     """Node for element wise UDF."""
 
-    func = Arg(callable)
-    func_args = Arg(tuple)
-    input_type = Arg(rlz.shape_like('func_args'))
-    _output_type = Arg(rlz.noop)
-
-    def __init__(self, func, args, input_type, output_type):
```

Comment:
By redefining the constructor none of the input arguments were validated.

Reply:
See https://github.com/ibis-project/ibis/blob/master/ibis/expr/signature.py#L180-L183